src/Keboola/GoogleSheetsWriter/Input/Paginator.php

```diff
@@ -33,16 +33,11 @@ public function pages(): \Generator
                 $i++;
             }
 
-            yield $values;
+            yield $this->offset => $values;
```

Comment:
Sorry, ale to mi nepřijde dobré. To je hrozně náhodné - proč je to klíč? Protože to zrovna jde poslat. Co když budeš posílat ještě "limit"? Nebál bych se úplně nového `PaginatorChunk`

Reply:
Jo, uz som mal spravenu triedu pre to `Page`, ale prislo mi, ze je to uplne zbytocna trieda. No skusil som to takto, zaujimalo ma, co na to povies ;)